lib/api/migrations/migration-1.0.1.js

```diff
@@ -25,7 +25,7 @@ function transformRequestParamsToString(mockedRequests) {
 
       // cancel previous default value
       if (mockedRequest.params === '{}') {
-        mockedRequest.params = null;
+        mockedRequest.params = undefined;
```

Comment:
I just saw that params is null, and not undefined if it doesn't exist
Not sure if this will have any affect on anybody